rules/windows/registry_event/sysmon_asep_reg_keys_modification.yml

```diff
@@ -4,35 +4,181 @@ description: Detects modification of autostart extensibility point (ASEP) in reg
 status: experimental
 references:
```

Comment:
Is this the reference?
https://www.microsoftpressstore.com/articles/article.aspx?p=2762082&seqNum=2
If yes, could you add it?

Reply:
It's not, I used a list from one of Microsoft webinars. But they seem to be identical. Will check Keys from the article and update if needed, thanks!

Reply:
Updated reference